File: 2. Loop Problems/Loop problems in python/p18.py
# // Write a python program to print all Prime numbers between 1 to n

import math


n = int(input("Enter Your Number: "))


for j in range(1, n + 1):

    if j <= 1:
        continue 

    elif j == 2 or j == 3:
        print(j, end=" ") 

    elif j % 2 == 0 or j % 3 == 0:
        continue  

    else:
        is_prime = True 
       
        for i in range(5, int(math.sqrt(j)) + 1, 6):
            if j % i == 0 or j % (i + 2) == 0:
                is_prime = False  # Not prime
                break

        if is_prime:
            print(j, end=" ")  # Print the prime number

File: 2. Loop Problems/Loop Problems in c/p18.c
// Write a C program to print all Prime numbers between 1 to n

#include<stdio.h>
#include<math.h>

int main () {
    int n;
    printf("Enter Your Number: ");
    scanf("%d",&n);

    for(int j=1;j<=n;j++) {

    if (j <= 1) {}

    else if (j == 2 || j == 3  ) {printf("%d ",j);}

    else if (j % 2 == 0 || j % 3 == 0) {}

    else {
    for (int i = 5;i <= sqrt(n);i+=6) {
         if (j % i == 0 || j % (i + 2) == 0) {   
                break; 
    }
    
    printf("%d ",j);}

    
}


}}